service/src/main/kotlin/fi/espoo/evaka/reservations/AttendanceReservationController.kt

```diff
@@ -894,9 +899,10 @@ SELECT
         SELECT jsonb_agg(jsonb_build_object(
             'date', ar.date,
             'startTime', ar.start_time,
-            'endTime', ar.end_time
+            'endTime', ar.end_time,
+            'staffCreated', eu.type = 'EMPLOYEE'
         ) ORDER BY ar.date, ar.start_time)
-        FROM attendance_reservation ar WHERE ar.child_id = p.id AND between_start_and_end(:dateRange, ar.date)
+        FROM attendance_reservation ar LEFT JOIN evaka_user eu ON ar.created_by = eu.id WHERE ar.child_id = p.id AND between_start_and_end(:dateRange, ar.date)
```

Comment:
created_by has not null so LEFT is unnecessary. would also add line breaks before join and before where

Reply:
Fixed!